atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransactionManager.java

```diff
@@ -195,7 +199,8 @@ protected SnapshotTransaction createTransaction(
                 timestampValidationReadCache,
                 lockAcquireTimeoutMs.get(),
                 getRangesExecutor,
-                defaultGetRangesConcurrency);
+                defaultGetRangesConcurrency,
+                sweepQueueWriter::enqueue);
```

Comment:
Why do we use a method reference here, where the constructor simply takes a `SweepQueueWriter`? (same below)